hyperspy/misc/utils.py

```diff
@@ -932,7 +935,7 @@ def closest_power_of_two(n):
 
 
 def stack(signal_list, axis=None, new_axis_name="stack_element", lazy=None,
-          show_progressbar=None, **kwargs):
+          copy_metadata=True, show_progressbar=None, **kwargs):
```

Comment:
What about ``stack_original_metadata`` instead?

Reply:
And keep the same behaviour regarding `metadata`?

Reply:
Sorry, I meant ``stack_metadata``

Reply:
Done.